lightning/src/ln/chan_utils.rs

```diff
@@ -581,14 +581,27 @@ impl ChannelTransactionParameters {
 		self.counterparty_parameters.is_some() && self.funding_outpoint.is_some()
 	}
 
-	/// Convert the owner specific parameters to owner independent parameters.
+	/// Convert the owner specific parameters to owner independent parameters,
+	/// given that the holder is the broadcaster.
 	///
 	/// self.is_populated() must be true before calling this function.
-	pub fn to_directed(&self, broadcaster_is_holder: bool) -> DirectedChannelTransactionParameters {
-		assert!(self.is_populated(), "self.late_parameters must be set before using to_directed");
+	pub fn to_directed_holder_is_broadcaster(&self) -> DirectedChannelTransactionParameters {
```

Comment:
Since this is a view, use "as" instead of "to" as per these [conventions](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv).

Perhaps the naming could be less verbose such as `as_holder_directed` since the return type should be responsible for describing what is meant by "directed".

Reply:
done